server/integration/feature-pack/feature-pack-build.xml

```diff
@@ -97,7 +97,22 @@
            template="configuration/examples/template-vault.xml"
            subsystems="configuration/examples/subsystems-node-auth.xml"
            output-file="docs/examples/configs/standalone-node-auth.xml"/>
-
+        <standalone
+           template="configuration/standalone/template.xml"
+           subsystems="configuration/examples/subsystems-memcached-local.xml"
+           output-file="docs/examples/configs/standalone-memcached.xml"/>
+        <standalone
+           template="configuration/standalone/template.xml"
+           subsystems="configuration/examples/subsystems-memcached-clustered.xml"
+           output-file="docs/examples/configs/clustered-memcached.xml"/>
+        <domain
+           template="configuration/domain/template.xml"
+           subsystems="configuration/examples/subsystems-memcached-domain.xml"
+           output-file="docs/examples/configs/domain-memcached.xml" />
+        <standalone
+           template="configuration/standalone/template.xml"
+           subsystems="configuration/examples/subsystems-websocket-local.xml"
+           output-file="docs/examples/configs/standalone-websocket-local.xml"/>
```

Comment:
This has the suffix -local while the memcached equivalent is only called standalone-memcached.xml